package.json

```diff
@@ -90,6 +90,7 @@
         "@nrwl/nx-cloud": "15.1.1",
         "@nrwl/tao": "15.8.5",
         "@nrwl/workspace": "15.8.5",
+        "@testing-library/cypress": "^9.0.0",
```

Comment:
`"9.0.0"` please use fix version, we use pin strategy for renovate

Reply:
ok, fixed